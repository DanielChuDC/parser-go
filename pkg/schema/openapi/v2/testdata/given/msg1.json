{
  "payload": {
    "type": "object",
    "nullable": true,
    "example": {
      "test": "example1"
    },
    "examples": [
      {
        "test": "example0"
      }
    ],
    "properties": {
      "test": {
        "type": "string",
        "nullable": true,
        "example": "test",
        "examples": [
          "fromTestArrayInProperties"
        ]
      },
      "test2": {
        "type": [
          "number"
        ],
        "nullable": true
      },
      "example": {
        "description": "this should not be affected"
      }
    },
    "additionalProperties": {
      "type": "number",
      "nullable": true,
      "example": "test2",
      "examples": [
        "fromTestArrayInAP"
      ]
    }
  }
}