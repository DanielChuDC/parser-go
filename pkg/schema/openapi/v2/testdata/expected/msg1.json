{
  "payload": {
    "additionalProperties": {
      "type": [
        "number",
        "null"
      ],
      "examples": [
        "fromTestArrayInAP",
        "test2"
      ]
    },
    "type": [
      "object",
      "null"
    ],
    "properties": {
      "example": {
        "description": "this should not be affected"
      },
      "test": {
        "type": [
          "string",
          "null"
        ],
        "examples": [
          "fromTestArrayInProperties",
          "test"
        ]
      },
      "test2": {
        "type": [
          "number",
          "null"
        ]
      }
    },
    "examples": [
      {
        "test": "example0"
      },
      {
        "test": "example1"
      }
    ]
  }
}